pinot-query-planner/src/main/java/org/apache/pinot/query/planner/logical/StagePlanner.java

```diff
@@ -41,12 +41,10 @@
  * This class is non-threadsafe. Do not reuse the stage planner for multiple query plans.
  */
 public class StagePlanner {
-  private final PlannerContext _plannerContext;
   private final WorkerManager _workerManager;
   private int _stageIdCounter;
 
   public StagePlanner(PlannerContext plannerContext, WorkerManager workerManager) {
-    _plannerContext = plannerContext;
```

Comment:
@walterddr specifically asked to keep this here in one of my previous PRs

Reply:
@walterddr can you comment?

Reply:
yes please don't remove planner context. it would be used for many other heuristics going forward even though it is not being used for now